### ISSUE TITLE
A-type natriuretic peptide level

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#4904]
**Preferred term label:**
'Abnormal circulating A-type natriuretic peptide level'/increased circulating A-type natriuretic peptide level/decreased circulating A-type natriuretic peptide level

**Synonyms**
Abnormal/increased.../decreased... atrial natriuretic peptide level

**Definition (free text, please give PubMed ID)**
A measurable change/increase/decrease in circulating levels of Atrial natriuretic peptide hormone, a protein which plays an important role in the regulation of body fluid volume and blood pressure. PMID:28552863

**Parent term (use hpo.jax.org/app)**
Abnormal circulating protein level (HP:0010876)/'Abnormal circulating A-type natriuretic peptide level'(new term)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
RGD:SJWang

**Comment by pnrobinson** [comment]
adding terms as HP_0020182-HP_0020184, they should be visible with the August release, thanks!
